src/webview/js/js.js

```diff
@@ -9,6 +9,7 @@ import type {
   WebViewInboundEventReady,
   WebViewInboundEventMessagesRead,
 } from '../generateInboundEvents';
+import { makeUserId } from '../../api/idTypes';
```

Comment:
We should run `tools/generate-webview-js` at this commit.

Reply:
Indeed, thanks for the catch!